pkg/core/tokens/default_signing_key.go

```diff
@@ -31,7 +31,11 @@ func (d *defaultSigningKeyComponent) Start(stop <-chan struct{}) error {
 	errChan := make(chan error)
 	go func() {
 		defer close(errChan)
-		if err := doWithRetry(ctx, d.createDefaultSigningKeyIfNotExist); err != nil {
+		backoff := retry.WithMaxDuration(10*time.Minute, retry.NewConstant(5*time.Second)) // if after this time we cannot create a resource - something is wrong and we should return an error which will restart CP.
+		err := retry.Do(ctx, backoff, func(ctx context.Context) error {
+			return retry.RetryableError(d.createDefaultSigningKeyIfNotExist(ctx)) // retry all errors
+		})
+		if err != nil {
```

Comment:
I was worried somewhere we might be checking if the `error` returned by `Do` is a `RetryableError` but it looks like we don't do that anywhere?